libs/components/split-view/src/lib/modules/split-view/split-view-drawer.component.ts

```diff
@@ -89,90 +95,104 @@ export class SkySplitViewDrawerComponent
+  #xCoord = 0;
+  #changeDetectorRef: ChangeDetectorRef;
+  #coreAdapterService: SkyCoreAdapterService;
+  #elementRef: ElementRef;
+  #splitViewMediaQuerySvc: SkySplitViewMediaQueryService;
+  #splitViewSvc: SkySplitViewService;
+  #windowRef: SkyAppWindowRef;
 
-  private xCoord = 0;
-
-  private _width: number;
+  #_width: number | undefined;
 
   constructor(
-    private changeDetectorRef: ChangeDetectorRef,
-    private coreAdapterService: SkyCoreAdapterService,
-    private elementRef: ElementRef,
-    private splitViewMediaQueryService: SkySplitViewMediaQueryService,
-    private splitViewService: SkySplitViewService
-  ) {}
+    changeDetectorRef: ChangeDetectorRef,
+    coreAdapterService: SkyCoreAdapterService,
+    elementRef: ElementRef,
+    splitViewMediaQuerySvc: SkySplitViewMediaQueryService,
+    splitViewSvc: SkySplitViewService,
+    windowRef: SkyAppWindowRef
+  ) {
+    this.#changeDetectorRef = changeDetectorRef;
+    this.#coreAdapterService = coreAdapterService;
+    this.#elementRef = elementRef;
+    this.#splitViewMediaQuerySvc = splitViewMediaQuerySvc;
+    this.#splitViewSvc = splitViewSvc;
+    this.#windowRef = windowRef;
+
+    this.splitViewDrawerId = `sky-split-view-drawer-${++skySplitViewNextId}`;
+  }
 
   public ngOnInit(): void {
-    this.setMaxWidth();
+    this.#setMaxWidth();
 
-    this.splitViewService.isMobileStream
-      .pipe(takeUntil(this.ngUnsubscribe))
+    this.#splitViewSvc.isMobileStream
+      .pipe(takeUntil(this.#ngUnsubscribe))
       .subscribe((mobile: boolean) => {
         this.isMobile = mobile;
-        this.changeDetectorRef.markForCheck();
+        this.#changeDetectorRef.markForCheck();
       });
   }
 
   public ngAfterViewInit(): void {
-    this.updateBreakpoint();
+    this.#updateBreakpoint();
   }
 
   public ngOnDestroy(): void {
-    this.ngUnsubscribe.next();
-    this.ngUnsubscribe.complete();
+    this.#ngUnsubscribe.next();
+    this.#ngUnsubscribe.complete();
   }
 
   public onResizeHandleMouseDown(event: MouseEvent): void {
     event.preventDefault();
     event.stopPropagation();
 
     if (
-      this.splitViewMediaQueryService.isWidthWithinBreakpiont(
+      this.#splitViewMediaQuerySvc.isWidthWithinBreakpiont(
         window.innerWidth,
         SkyMediaBreakpoints.xs
       )
     ) {
       return;
     }
 
-    this.setMaxWidth();
-    this.isDragging = true;
-    this.xCoord = event.clientX;
+    this.#setMaxWidth();
+    this.#isDragging = true;
+    this.#xCoord = event.clientX;
 
-    this.coreAdapterService.toggleIframePointerEvents(false);
+    this.#coreAdapterService.toggleIframePointerEvents(false);
 
     fromEvent(document, 'mousemove')
       .pipe(
         takeWhile(() => {
-          return this.isDragging;
+          return this.#isDragging;
         })
       )
-      .subscribe((moveEvent: any) => {
-        this.onMouseMove(moveEvent);
+      .subscribe((moveEvent) => {
+        this.onMouseMove(moveEvent as MouseEvent);
```

Comment:
The `fromEvent` operator accepts generics, so you can do:
```
fromEvent<MouseEvent>(...
```

Reply:
Fixed.